xbmc/network/WebServer.cpp

```diff
@@ -102,9 +102,23 @@ CWebServer::CWebServer()
 #endif
 }
 
+static MHD_Response* create_response(size_t size, void* data, int free, int copy)
+{
+#if (MHD_VERSION >= 0x00094001)
+  MHD_ResponseMemoryMode mode = MHD_RESPMEM_PERSISTENT;
+  if (copy)
+    mode = MHD_RESPMEM_MUST_COPY;
+  else if (free)
+    mode = MHD_RESPMEM_MUST_FREE;
+  return MHD_create_response_from_buffer(0, nullptr, mode);
+#else
+  return MHD_create_response_from_data(0, nullptr, free, copy);
```

Comment:
Doh. Please fix i am not around a puter for a few days..
24. jun. 2016 17:16 skrev "popcornmix" <notifications@github.com>:

> In xbmc/network/WebServer.cpp
> <https://github.com/xbmc/xbmc/pull/10023#discussion_r68413430>:
>
> > @@ -102,9 +102,23 @@ CWebServer::CWebServer()
> >  #endif
> >  }
> >
> > +static MHD_Response* create_response(size_t size, void* data, int free, int copy)
> > +{
> > +#if (MHD_VERSION >= 0x00094001)
> > +  MHD_ResponseMemoryMode mode = MHD_RESPMEM_PERSISTENT;
> > +  if (copy)
> > +    mode = MHD_RESPMEM_MUST_COPY;
> > +  else if (free)
> > +    mode = MHD_RESPMEM_MUST_FREE;
> > +  return MHD_create_response_from_buffer(0, nullptr, mode);
> > +#else
> > +  return MHD_create_response_from_data(0, nullptr, free, copy);
>
> This should be:
> return MHD_create_response_from_data(size, data, free, copy);
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbmc/xbmc/pull/10023/files/8739447920628b1d0ee1ceabf223d649a5b4a4aa..95e689259e2ce5ac47dd91853f428f8552f13268#r68413430>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFktYOKfZ-ZI878-YIrWPL887znJlR-nks5qO_TBgaJpZM4I8fi7>
> .
>